### PR TITLE
added jenkinsfile

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                bat 'pip3 install -r requirements.txt'
+                bat 'pip install allure-behave'
+            }
+        }
+        stage('UNIT TEST') {
+            steps {
+                bat 'pytest test'
+            }
+        }
+        stage('TEST') {
```
Comment (Enrique-C) [inline Jenkinsfile]: It would be API TEST

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            parallel {
+                stage('Trello') {
+                    steps {
+                            bat 'behave -f allure_behave.formatter:AllureFormatter -o reports trello/api/features/ --tags=~@defect'
+                    }
+                }
+                stage('Pivotal') {
```
Comment (Enrique-C) [inline Jenkinsfile]: It would be Pivotal Tracker

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                    steps {
+                            bat 'behave -f allure_behave.formatter:AllureFormatter -o reports pivotal/api/features/ --tags=~@defect'
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (Enrique-C) [inline Jenkinsfile]: Add a code line at the last

File: trello/api/features/Card.feature [hunk 4]
```diff
@@ -308,44 +308,6 @@ Feature: Card
     And Should return status code 200
 
 
-  @Functional
```
Comment (Enrique-C) [inline trello/api/features/Card.feature:311]: What happens here?

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by Enrique-C — APPROVED** [review]

**Pull request merged**